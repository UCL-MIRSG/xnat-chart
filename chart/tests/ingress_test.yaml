templates:
  - templates/ingress.yaml
tests:
  - it: ingress manifest should match snapshot
    set:
      web.ingress.enabled: true
    asserts:
      - matchSnapshot:
          path: spec
