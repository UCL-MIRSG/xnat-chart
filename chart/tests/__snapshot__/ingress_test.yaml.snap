ingress manifest should match snapshot:
  1: |
    ingressClassName: nginx
    rules:
      - host: chart-example.local
        http:
          paths:
            - backend:
                service:
                  name: RELEASE-NAME-xnat-headless
                  port:
                    number: 80
              path: /
              pathType: Prefix
